foldamers/cg_model/cgmodel.py

```diff
@@ -244,6 +248,125 @@ def export(self, filename):
         pickle.dump(self, pickle_out)
         pickle_out.close()
 
+    def _validate_bonded_forces(self):
+
+        # check the names that are included in the dictionaries to make sure
+        # there are no mispellings.
+
+        # dictionary of the force attributes
+
+        # for each force attribute, which appears in the dictionary defining the forces,
+        # define certain properties;
+        #     "default name" : the name to look for default definitions of those nonboded forces.
+        #     "default value" : the value to store if the default is not given,
+        #     "suffix" : the suffix that those forces should have
+        # We are trying to minimize the number of places adding new forces changes the code,
+        # and this should help with that.
+
+        self.bonded_force_attributes = {
+            "bond_lengths": {
+                "default_name": "default_bond_length",
+                "default_value": self.default_length,
+                "suffix": "bond_length",
+            },
+            "bond_force_constants": {
+                "default_name": "default_bond_force_constant",
+                "default_value": self.default_bond_force_constant,
+                "suffix": "bond_force_constant",
+            },
+            "equil_bond_angles": {
+                "default_name": "default_equil_bond_angle",
+                "default_value": self.default_angle,
+                "suffix": "equil_bond_angle",
+            },
+            "bond_angle_force_constants": {
+                "default_name": "default_angle_bond_force_constant",
```

Comment:
default_name should be "default_bond_angle_force_constant"?

Reply:
Yep.